sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeOperationResult.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Collection of <see cref="JobMetadata"/> objects corresponding
+    /// to a batch of documents, and information about the batch operation.
+    /// </summary>
+    public class AnalyzeOperationResult
```

Comment:
any reason why you are not exposing the properties for the `TaskState` and `TasksState` ?

Reply:
I am not sure if it is required for a user, we can expose them. But I don't think the neccessity.

Reply:
If the service is exposing them, I am curious to know why the SDK isn't.
As a user, I can see how knowing how many of the tasks failed vs completed is interesting. Also the name of the job

Reply:
can happen in a future PR